enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1847,6 +1847,7 @@ type cdcWriter struct {
 	firstChunk []byte
 	fileType   rfpb.FileMetadata_FileType
 
+	mu sync.Mutex // protects writtenChunks, numChunks, firstChunk, fileType
```

Comment:
This might be a personal preference, but I like to organize struct fields such that protected variables are below the mutex that protects them, i.e.

```
   eg *errgroup.Group

    mu sync.Mutex
	writtenChunks []*rspb.ResourceName
	numChunks  int
	firstChunk []byte
	fileType   rfpb.FileMetadata_FileType
```

Though you have a comment which accomplishes the same goal.

Reply:
+1

Reply:
Done